packages/expo-dev-client/metro.config.js

```diff
@@ -0,0 +1,3 @@
+const { createMetroConfiguration } = require('expo-yarn-workspaces');
```

Comment:
should this file be gitignored?

Reply:
I don't think so. However, maybe it should be npmingore.